src/TicketContext.tsx

```diff
@@ -170,22 +217,28 @@ const TicketContextProvider: React.FC = ({children}) => {
   );
 
   useInterval(
-    updateFareContracts,
+    updateFareContractTickets,
     1000,
     [],
     !activeReservations.some((res) => res.paymentStatus === 'CAPTURE'),
   );
 
   useEffect(() => {
-    updateFareContracts();
+    updateFareContractTickets();
+  }, []);
+
+  useInterval(updateFareContractTypes, 10000);
```

Comment:
Refreshing every ten minute. Or does it need to be refreshed at a set interval at all?

Reply:
Haven't read/seen any of the code but just a heads up that you can look into: I think there are limitations on timeout time for android. Around 2 minutes iirc

Reply:
I see now that I wrote 10 minutes, it is actually set to 10 seconds, which probably is unnecessary often.

Reply:
We should probably retrieve this on app startup instead. This shouldn't update that often?